openshift/masu-template.yaml

```diff
@@ -237,10 +237,10 @@ objects:
           livenessProbe:
             failureThreshold: 3
             httpGet:
-              path: /api/v1/status
+              path: /api/v1/status/?liveness
```

Comment:
This is interesting.  Is the `?` standard for liveliness checks?

Reply:
The `?` just denotes URI query parameters.  (See: RFC3986 & friends)

I was mostly just being lazy and reusing the existing status endpoint over creating an all-new endpoint just to have it output a simple string.

Typically, liveness probes are supposed to be low overhead to just show "early signs of life" within the app. (e.g. a TCP connect to a port or checking for a particular PID) For a Flask app, we start up fast enough, that I can keep the check as an HTTP check that isn't as heavyweight as the status is becoming. 

Adding the checks for Celery involves round-trip communications with RabbitMQ and the Celery workers. So, just checking `/status` now is too slow to be appropriate for a liveness check.

Reply:
ok cool, thanks for confirming.  Yeah I knew about `?` for query, thats what threw me off.  Probably not a big deal, but could we make this a new endpoint instead?

Reply:
Can we?  Sure. There's lots of ways to skin this cat. :smiley_cat: 

I think my main question is, what does it buy us?

Reply:
ha I don't know.  It just is a non-standard use of `?`, but we can keep it.